src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -938,6 +942,34 @@ static int32_t cc_set(InteractiveConsole& console, const utf8** argv, int32_t ar
             }
             console.Execute("get cheat_disable_support_limits");
         }
+        else if (strcmp(argv[0], "current_rotation") == 0 && invalidArguments(&invalidArgs, int_valid[0]))
+        {
+            uint8_t currentRotation = get_current_rotation();
+            rct_window* mainWindow = window_get_main();
+            // as there are only 4 possible rotations, make any input go to 0-3 to avoid unneccessary work
+            int32_t newRotation = int_val[0] % 4;
```

Comment:
Looks alright to me.

Reply:
I meant if you passed a value like 20 then it would otherwise go through 5x ish the iterations necessary. But really it doesn't make sense to accept values outside the 0-3 range so I should probably just print a message saying the accepted range instead?

Reply:
That sounds fine, but you need to limit it to [0, 3] range first. So far you only limited it to [-3, 3].